### ISSUE TITLE
accessibility not working 

### ISSUE THREAD

**Issue by harsha1439** [srk-sharingan/sharingan-rn-modal-dropdown#38]
Is this package compatible to accessibility? If yes, I have few issues coming up while testing accessibility.

Scenario 1: when screen mounted with textInputPlaceholder={'Sharing-an Drop down'}, it's reading out accessibility label.
But when compared to my custom drop down It lacks the arrow accessibility. This is some how reading, but scenario 2 accessibility is completely not working with sharing-an dropdown
https://user-images.githubusercontent.com/12248640/133320559-5ca2c1a7-225a-41a7-a60a-da636cebb03d.mov

Scenario2: when I pre populate data it's not reading anything, but my custom dropdown is reading the data. screen recording uploaded.
https://user-images.githubusercontent.com/12248640/133320978-1f27b1d1-8f8e-467a-a68a-9533e0020256.mov

Can you see scenario 2, it just skips without reading any accessibility in sharing-an dropdown. 

**Expected behavior**
We will have one item selected dynamically from dropdown when the page loads. When this scenario is tested it skips the whole box accessibility readout and that is the major issue. If this package is accessibility compatible, let me know if i'm missing anything. 

 - Device: iphone 11 pro max
 - OS: iOS 14

**Comment by selvamk-js** [comment]
Nope, this package does not provide support for any accessibility. Maybe in the next upcoming release, I will try to investigate and do the necessary actions. Thanks
